docs/arcaflow/workflows/schemas.md

```diff
@@ -0,0 +1,63 @@
+# Workflow Schemas
+
+## Schema Names
+
+### Scalar Names
+
+Schemas that are not composed within an ObjectSchema do not have an `Object ID`. They use a stringified version of their `TypeID` for their schema name.
+
+| schema        |     type id                 |
+|---------------|-----------------------------|

```

Comment:
Unless there is good reason to the contrary, the column headings should be consistent across the four tables in this file.  That is, I _think_ each of these tables is relating one or more schemas to a _name_.  So, the second column here should be headed "name" (even though the entries in the column are actually type IDs -- they are also the schema _names_).

Similarly, at line 57, the heading should be "name", not "new name".  (The name doesn't change (i.e., become "new") given the same first and second schema values.)

Reply:
did it